src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -0,0 +1,84 @@
+
+<figcaption>
+  Click the **Query your data** button to open the query builder and start writing a NRQL query. Once you've written a valid query, you can create an alerts condition from it.
+</figcaption>
+
+The condition shows each host’s CPU utilization. Use this NRQL query, or something similar that's relevant to what you're monitoring, to create an alerts condition.
+
+![A screenshot of a NRQL query that finds the average CPU utilization faceted by hostname.](./images/nrql-query.png 'A screenshot of a NRQL query that finds the average CPU utilization faceted by hostname.')
+
+<figcaption>
+  This query finds the average CPU utilization from a system and then breaks it
+  down (facets) by individual host names.
+</figcaption>
+
+Once you've created a valid NRQL query, click the **Create alert** button.
+
+Learn more about the [NRQL syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/#syntax) to create a variety of queries.
+
+## Step 2: Name and revise your NRQL condition [#step2-nrql]
+
+Give your NRQL alerts condition a name that's meaningful to you. Your query's visualization is shown automatically.
+
+![The query's visualization shows each host’s average CPU utilization.](./images/show-nrql-graph.png 'The query's visualization shows each host’s average CPU utilization.')
+
+<figcaption>
+  The query's visualization shows each host’s average CPU utilization. A critical violation will open if any host’s average CPU utilization is over 70%. A warning threshold is met at 67%.
+
+  Warning thresholds are described below.
+</figcaption>
+
+## Step 3: Set your condition thresholds [#step3-nrql]
+
+Set critical and warning thresholds to determine when you’re notified about your system's performance. 
+
+In this example, our critical threshold (red) checks for any hosts with CPU utilization over 70% for at least 5 minutes. A warning threshold (yellow) is also added showing when any host's CPU utilization goes over 67% for at least 5 minutes.
+
+![A visualization of the NRQL query with critical and warning threshold lines.](./images/nrql-set-threshold.png 'A visualization of the NRQL query with critical and warning threshold lines.')
+
+<figcaption>
+  Use the **Set your condition thresholds** section to set the critical and warning thresholds for your condition.
+</figcaption>
+
+At least one critical threshold is required for your alerts condition. Warning thresholds are optional. 
+
+Warning thresholds don't create an incident or notify you like the critical threshold. If a critical threshold opens an incident and notifies you, warning threshold violations will be included in that report.
+
+## Step 4: Create a policy
+
+Finally, you can add your condition to an existing policy or create a new one.
```

Comment:
Saying "you can" makes this sound optional, but it is mandatory. Maybe "you must," or just "add your condition... etc."

Reply:
Excellent point